src/viewer.js

```diff
@@ -624,13 +623,12 @@ class VolumeImageViewer {
                   blendingInformation: blendingInformation
                 }
               } else {
-                const maxValue = Math.pow(2, bitsAllocated)
                 const defaultBlendingInformation = new BlendingInformation({
                   opticalPathIdentifier: `${opticalPathIdentifier}`,
-                  color: [...colormap[i % colormap.length]],
-                  opacity: 1.0,
-                  thresholdValues: [0, maxValue],
-                  limitValues: [0, maxValue],
+                  color: [...colormap[i % colormap.length]].slice(0,3),
+                  opacity: 1,
+                  thresholdValues: [0, 255],
+                  limitValues: [0, 255],
```

Comment:
We need to use `maxValue` to work correctly with 16-bit images.

Reply:
@hackermd uhm the data in test case (screenshot in https://github.com/MGHComputationalPathology/dicom-microscopy-viewer/pull/69#issue-777473194) are 16 bits (i.e. bitsAllocated = 16), but maybe then the actaul values are saved as 8 bit. That may be the issue there. I will check the actual dataarray of the test data.

Reply:
yes the data form the test case have the metadata wrong (16 bits), but then the image is actaully 8 bits. 
So I will do:
1) put a check in decodeFrame that checks the maximum of the array (if the array is 255, but bitsAllocated = 16, it will return bitsPerSample = 8 instead of 16), 
2) I will update this PR to use then the max 8/16bits value

Reply:
uhm no (1) it is not possible, this options are set before retrieve the data array. So we need that the metadata are provided properly. I will just rool back to use the max 8/16bits value. Still this PR fixes some bugs.

Reply:
We shouldn't do such hacks! The viewer should assume that the metadata is correct. If it isn't the data set should be fixed.

Reply:
I agree, the app should not fix errors in the  metadata. In fact, I have already revered all the changes for (1). This PR at the moment fix only point (2).